### PR TITLE
[issue 206] Post release improvements

### PR DESCRIPTION
**Pull request by parente**
* README updates
* Log a message on server start

### CODE REVIEW

**Comment by parente** [comment]
@jhpedemonte, can you review and merge?

File: README.md [hunk 1]
```diff
@@ -32,25 +32,22 @@ The behavior of the application is similar to that of [Thebe](https://github.com
...
 
-If you want to see the dashboard server in action, you can run a handful of demos we included in this project in a Docker container. After setting up Docker (e.g. using [docker-machine](https://docs.docker.com/machine/get-started/)), do the following in a git clone of this repo:
+You can install the dashboard server using `npm`. See the next section about how to install and configure the other prerequisite components.
 
 ```
-make build
-make examples
-make demo-container
+npm install -g jupyter-dashboards-server
+jupyter-dashboards-server --help
```
Comment (jhpedemonte) [inline README.md]: `--help` just spits out the config file, but doesn't tell user how to specify those settings when starting on the command line.

Comment (parente) [inline README.md]: It does at the top. I can add examples to the bottom.

Comment (parente) [inline README.md]: Done.

**Pull request merged**